### PR TITLE
Prepare spring cloud azure 4.5.0 release

### PR DESCRIPTION
**Pull request by Netyyyy**
As the title to prepare Spring Cloud Azure 4.5.0 release

### CODE REVIEW

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.azure.spring</groupId>
   <artifactId>spring-cloud-azure-dependencies</artifactId>
-  <version>4.5.0-beta.2</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-dependencies;current} -->
```
Comment (saragluna) [inline sdk/boms/spring-cloud-azure-dependencies/pom.xml:9]: the azure-sdk bom version in this file is not correct.

File: sdk/spring/spring-cloud-azure-service/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 4.5.0-beta.2 (Unreleased)
+## 4.5.0 (2022-12-05)
+
+Please refer to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-java/blob/1f6dc567b80b66f80ef53933ab8473cedbe4c2dc/sdk/spring/CHANGELOG.md#450-2022-12-05) for more details.
```
Comment (saragluna) [inline sdk/spring/spring-cloud-azure-service/CHANGELOG.md:5]: If we change this link to https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/CHANGELOG.md#450-2022-12-05, will it break the link check?

File: sdk/spring/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.5.0-beta.2 (Unreleased)
```
Comment (saragluna) [inline sdk/spring/CHANGELOG.md:3]: Could you help add a section for something like this (I copied this from 4.5.0-beta.1 changelog, but we need to change this for 4.5.0)

```
### Spring Cloud Azure Starter
This section includes changes in `spring-cloud-azure-starter` module.
#### Features Added
+ Support Azure hosted PostgreSQL and MySQL services authenticating with Azure AD [#30024](https://github.com/Azure/azure-sdk-for-java/pull/30024).
```

which means we are GAing these two libs.

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml (hunk 1 above)
Comment (saragluna) [reply sdk/boms/spring-cloud-azure-dependencies/pom.xml:9]: Could you help check whether we have an action for upgrading azure-sdk-bom in our release todo list?

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml (hunk 1 above)
Comment (Netyyyy) [reply sdk/boms/spring-cloud-azure-dependencies/pom.xml:9]: Not find it, will add one.

File: sdk/spring/spring-cloud-azure-service/CHANGELOG.md (hunk 2 above)
Comment (Netyyyy) [reply sdk/spring/spring-cloud-azure-service/CHANGELOG.md]: Not break check in this PR

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml [hunk 4]
```diff
@@ -51,7 +51,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-sdk-bom</artifactId>
-        <version>1.2.6</version>   <!-- NOTE: This should be updated manually. -->
```
Comment (saragluna) [inline sdk/boms/spring-cloud-azure-dependencies/pom.xml:54]: Still not correct. The latest version from maven central is [1.2.8](https://mvnrepository.com/artifact/com.azure/azure-sdk-bom). Could you help update the version and compare the `azure-core` version used in spring-cloud-azure-core and the one in `azure-sdk-bom`, to see whether they are the same?

File: sdk/spring/CHANGELOG.md [hunk 5]
```diff
@@ -1,9 +1,16 @@
...
+## 4.5.0 (2022-12-05)
 Upgrade Spring Boot dependencies version to 2.7.6 and Spring Cloud dependencies version to 2021.0.5
 Upgrade Spring Boot dependencies version to 2.7.4 and Spring Cloud dependencies version to 2021.0.4
 
+### Spring Cloud Azure Starter
+This section includes changes in `spring-cloud-azure-starter` module.
+
+#### Features Added
+- GA the `spring-cloud-azure-starter-jdbc-mysql`. This starter supports Azure hosted MySQL service authenticating with Azure AD[#30024](https://github.com/Azure/azure-sdk-for-java/pull/30024).
+- GA the `spring-cloud-azure-starter-jdbc-postgresql`. This starter supports Azure hosted PostgreSQL service authenticating with Azure AD[#30024](https://github.com/Azure/azure-sdk-for-java/pull/30024).
```
Comment (saragluna) [inline sdk/spring/CHANGELOG.md:12]: Is the PR number correct? If not, maybe we should just skip the PR number.

File: sdk/spring/spring-cloud-azure-actuator/CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 4.5.0-beta.2 (Unreleased)
+## 4.5.0 (2022-12-05)
+
+Please refer to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/CHANGELOG.md#450-2022-12-05) for more details.
```
Comment (saragluna) [inline sdk/spring/spring-cloud-azure-actuator/CHANGELOG.md:5]: Maybe we should always use the main branch for the changelog.

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml (hunk 4 above)
Comment (Netyyyy) [reply sdk/boms/spring-cloud-azure-dependencies/pom.xml:54]: `azure-core` in `spring-cloud-azure-core` is 1.34.0 and in `azure-sdk-bom` is 1.33.0

**Comment by saragluna** [comment]
Use https://github.com/Azure/azure-sdk-for-java/pull/32464 instead.

**Pull request closed without merging**